src/content/defi/index.md

```diff
@@ -0,0 +1,245 @@
+    "Probably the biggest use case for Ethereum right now.",
+    "Powered by smart contracts on the Ethereum blockchain.",
+  ]
+---
+
+_Defi is an alternative to a financial system that many of us have grown to mistrust thanks to the cycles of mismanagement, crashes, and bail outs. It's an open global system that gives you more control and visibility over your money. It's exposure to global markets when your currency at home is failing. And there are no banker bonuses – it's largely owned by its users or maintained as a public good. And right now there's over 40 billion dollars worth of crypto in defi applications._
+
+## What's defi?
+
+Defi is a collective term for products that are making financial products and services accessible to anyone who can use Ethereum – anyone with an internet connection. With defi, there are no centralized authorities who can block payments or deny you access to anything.
+
+There's a booming crypto economy out there, where you can lend, borrow, long/short, earn interest, and more. Crypto-savvy Argentinians have used defi to escape crippling inflation. Companies have started streaming their employees their wages in real time. Some folks have even taken out and paid off loans worth millions of dollars without the need for any personal identification.
+
+## Defi vs traditional finance
+
+One of the best ways to see the potential of defi is to understand the problems that exist today.
+
+- Some people aren't granted access to set up a bank account or use financial services
+- Financial services can block you from getting paid
+- The hidden charge of financial services is your personal data
+- Governments can close down markets at will
+- Money transfers can take days due to internal human processes
+- There's a premium to financial services because intermediray institutions need their cut
+
+| Defi                        | Traditional finance                                      |
+| --------------------------- | -------------------------------------------------------- |
+| You control your funds      | Your funds are looked after by companies                 |
+| Transfers happen in minutes | Payments can take days due to manual processes           |
+| Pseudonymous                | Financial activity is tightly coupled with your identity |
+| Open to anyone              | You must apply to use financial services                 |
+
+<ButtonLink to="/dapps/?category=finance">Explore defi apps</ButtonLink>
+
+## What can you do with defi?
+
+There's a decentralized alternative to most financial services. But Ethereum also creates opportunities for creating financial products that are completely new. This is an ever-growing list.
+
+- [Send money around the globe](#send-money)
+- [Stream money around the globe](#stream-money)
+- [Access stable currencies](#stablecoins)
+- [Lend/borrow funds](#lending)
+- [Borrow without collateral](#flash-loans)
+- [Start crypto savings](#savings)
+- [Swap tokens](#swaps)
```

Comment:
I don't think we need to differentiate between "swap" & "trade" - aren't those essentially the same? I would advocate for "trade", as that's the term more broadly used. I feel like only our industry uses the term "swap"... not sure why.

Reply:
Yeah, "swap" seems to be common in the community, but new users likely don't think that way, they think about "trading"

Reply:
Yeah I understand. In my head the distinction between swaps and trades is like uniswap vs dydx.

To me dydx is super intimidating (trading for leverage etc) whereas Uniswap is "ooo I need to grab some Dai, I'll swap out some of my ETH"....

Not sure whether this distinction is relevant for our newbie audience...